packages/docusaurus/lib/default-theme/DocBody/index.js

```diff
@@ -13,6 +13,22 @@ import Head from '@docusaurus/Head';
 
 import styles from './styles.module.css';
 
+const Headings = ({headings}) => {
+  if (!headings.length) return null;
+  return (
+    <ul className="contents contents__left-border">
```

Comment:
I see why the left border is duplicated now - you're using recursion. Leave it like this first I'll fix on Infima end and update here.

Reply:
Or maybe you can render `contents__left-border` if depth === 1

Reply:
I removed `depth` and set a `isChild` props. Anyway, Its fixed now and will be waiting for infima update